lua/lspconfig/graphql.lua

```diff
@@ -8,7 +8,7 @@ configs[server_name] = {
   default_config = {
     cmd = { bin_name, 'server', '-m', 'stream' },
     filetypes = { 'graphql' },
-    root_dir = util.root_pattern('.git', '.graphqlrc'),
+    root_dir = util.root_pattern('.git', '.graphqlrc*'),
```

Comment:
Seems like it would make sense to add `graphql.config.*` as well according to the link the pull request description.